docker-compose.yml.erb

```diff
@@ -36,11 +37,26 @@ services:
     expose:
       - 80
       - 443
+    networks:
+      default:
+        aliases:
+          - "<%= config.hostname %>"
     # Please define ports in your docker-compose.override.yml
     # ports:
     #  - '80:80'
     #  - '443:443'
 
+  setup:
+    image: quay.io/organisation/<%= config.name %>:dev
+    depends_on:
```

Comment:
Does it really? 🤔

Reply:
I guess not but it was a shorthand for the transitive dependency on the database and other supporting services.
We could duplicate?